9.2/root/usr/bin/run-postgresql

```diff
@@ -16,5 +16,10 @@ fi
 
 set_passwords
 
+if [ -f ${CONTAINER_SCRIPTS_PATH}/post-init.sh ]; then
+  log_info 'Sourcing post-init.sh ...'
+  source ${CONTAINER_SCRIPTS_PATH}/post-init.sh
```

Comment:
don't need to be source, but I inspired from mySQL https://github.com/openshift/mysql/blob/master/5.6/root/usr/bin/run-mysqld#L30